lib/gds_api/test_helpers/publishing_api.rb

```diff
@@ -11,15 +11,15 @@ module PublishingApi
 
       PUBLISHING_API_ENDPOINT = Plek.current.find('publishing-api')
 
-      def stub_publishing_api_unreserve_path(base_path, publishing_app)
+      def stub_publishing_api_unreserve_path(base_path, publishing_app = /.*/)
```

Comment:
Hmm, I wonder for the amount of times we are actually going to use these stubs if it would be simpler and easier to understand to just require a `publishing_app` instead of using a regex?

Reply:
Originally I didn't default it, but then I found that it bloats the test code in Content Publisher, where most of the time we don't care about the publishing_app, and it was easier to do this than adding another 'any'-type stub.

Reply:
Okay sure!